bin/index.js

```diff
@@ -142,13 +142,10 @@ const addExchangeUrl = (manifest) => {
 
           if (
             !input.match(
-              /^https:\/\/www\.adobeexchange\.com\/experiencecloud\.details\..+\.html$/gi
+              /^https?:\/\/[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]+\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)$/gi
             )
           ) {
-            return (
-              'Must match the pattern ' +
-              '"https://www.adobeexchange.com/experiencecloud.details.######.html".'
-            );
+            return 'Must be an URL.';
```

Comment:
Technically, it's "a URL". English it weird, but if a/an comes before an acronymn, then you base a/an off how the acronym sounds rather than how it's spelled. So even though it starts with a U, which is a vowel, you pronounce it by starting with a "Y" sound, which is a consonant, so "a URL". 🤷‍♂️ I don't make the rules.

Reply:
Actually, this rule doesn't apply only to acronyms, but all words. For example, "a university" rather than "an university".

Reply:
Thanks for the free lesson. Changed it.